src/components/NavBar/NavBar.js

```diff
@@ -25,18 +22,10 @@ const NavBar = () => {
                 <div className="element-container resource-btn">
                     <DropDownButton
```

Comment:
great work, do you mind checking out the DropDownButton component as well and then 
- remove the arrow attached to it
- rename the component to `ResourcesButton` since it's not a dropdown anymore lol